lib/matplotlib/backends/backend_ps.py

```diff
@@ -1440,27 +1440,32 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
```

Comment:
Hmm, I don't see where `_get_shell_cmd` is used at all in the whole matplotlib. I don't see how to use that here to change this command string to a list.